docs/build-logs.md

```diff
@@ -69,6 +69,12 @@ Please use the "Validate Elasticsearch configuration" to verify the setup.
    alt="Configuration - Storing Jenkins Pipeline Logs in Elastic visualizing logs in Kibana and through Jenkins"
    src="./images/jenkins-config-elastic-logs-with-visualization-through-jenkins.png" />
 
+#### Log Exporter Configuration
+
+* To export logs to an OTLP endpoint, configure the following environment variables:
+- `otel.logs.exporter=otlp` : Specifies that logs should be exported using the OTLP exporter.
+- `otel.logs.mirror_to_disk=true` : Ensure logs are written to disk, making them visible
```

Comment:
already documented at https://github.com/jenkinsci/opentelemetry-plugin/blob/main/docs/build-logs.md#can-build-logs-be-saved-in-the-build-folder-locally